Services/WindowServer/WindowManager.h

```diff
@@ -169,8 +169,21 @@ class WindowManager : public Core::Object {
     void start_window_resize(Window&, const Gfx::IntPoint&, MouseButton);
     void start_window_resize(Window&, const MouseEvent&);
 
-    const Window* active_fullscreen_window() const { return (m_active_window && m_active_window->is_fullscreen()) ? m_active_window : nullptr; }
-    Window* active_fullscreen_window() { return (m_active_window && m_active_window->is_fullscreen()) ? m_active_window : nullptr; }
+    const Window* active_fullscreen_window() const
+    {
+        if (m_active_window && m_active_window->is_fullscreen()) {
+            return m_active_window;
+        }
+        return {};
+    };
+
+    Window* active_fullscreen_window()
+    {
+        if (m_active_window && m_active_window->is_fullscreen()) {
+            return m_active_window;
+        }
+        return {};
```

Comment:
doesn't this function return a regular pointer? why are we using `{}` instead of `nullptr`?

Reply:
Yes. I will change that.

The reason for the initial change was that the conversion operator from `std::nullptr_t` to `WeakPtr<T>` is removed so when the `?:` operator is used and returns different types (not convertible) the compiler complains. Note, the `m_active_window` is a `WeakPtr<Window>` and `nullptr` is a `std::nullptr_t`.

Reply:
@ldm5180: We can also just leave that file alone, there is no need to rewrite these functions without the ternary operator.

Reply:
Same as above comment.